### PR TITLE
Add the export of catalog into Magento

### PR DESCRIPTION
**Pull request by sebastienbeau**
This PR depend of this one
[ ] https://github.com/OCA/connector-magento/pull/43

### CODE REVIEW

File: magentoerpconnect_catalog/__openerp__.py [hunk 1]
```diff
@@ -25,26 +25,44 @@
  'version': '2.0.0',
  'category': 'Connector',
  'depends': ['magentoerpconnect',
-             'product_links',
-             'product_images',
+             'product_custom_attributes',
+             #'product_links',
```
Comment (rvalyi) [inline magentoerpconnect_catalog/__openerp__.py:29]: is dropping product_links module wanted?

File: magentoerpconnect_catalog/__openerp__.py [hunk 2]
```diff
@@ -25,26 +25,44 @@
...
+             'product_image',
+             'product_category_image',
              ],
  'author': 'MagentoERPconnect Core Editors',
  'license': 'AGPL-3',
- 'website': 'http://www.openerp-magento-connector.com',
+ 'website': 'https://launchpad.net/magentoerpconnect',
```
Comment (rvalyi) [inline magentoerpconnect_catalog/__openerp__.py:35]: gah still not the correct one...

File: magentoerpconnect_catalog/__openerp__.py [hunk 3]
```diff
@@ -25,26 +25,44 @@
...
 
+   - to be used, you need to manually create an attribute set which match with magento 'Default' attribute set (generally magento_id '4')
+   
+* product image: dependency
+
+   - dev branch: lp:~akretion-team/openerp-product-attributes/openerp-product-attributes-image
+   - future production branch: lp:openerp-product-attributes/openerp-product-attributes
+
+TODO:
+* import/export product links
```
Comment (rvalyi) [inline magentoerpconnect_catalog/__openerp__.py:55]: hum that may explain why product_links commented for now...

**Comment by sebastienbeau** [comment]
This project will be not merge on 7 use the project here https://github.com/akretion/connector-magento-catalog

**Pull request closed without merging**